flake.nix

```
@@ -0,0 +1,139 @@
+{
+  description =
+    "Quarkdown CLI - A modern Markdown-based typesetting system with superpowers";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    flake-utils.url = "github:numtide/flake-utils";
+  };
+
+  outputs = { self, nixpkgs, flake-utils }:
+    flake-utils.lib.eachDefaultSystem (system:
+      let
+        pkgs = nixpkgs.legacyPackages.${system};
+
+        # Quarkdown version - using latest stable release
+        version = "1.4.0";
+
+        # Build Quarkdown from source
+        quarkdown = pkgs.stdenv.mkDerivation rec {
+          pname = "quarkdown";
+          inherit version;
+
+          src = pkgs.fetchFromGitHub {
+            owner = "iamgio";
+            repo = "quarkdown";
+            rev = "v${version}";
+            sha256 = "sha256-9WQ3+VjlZmlDcHwSVOWD/ZGJfRCnXdKF0humGq2d8CU=";
+          };
```

```suggestion
          src = ./.;
```

I don't think the flake should be re-fetching sources, by the time the flake is running this flake file will already have been located and the sources can be used from the same location (whether that is locally with `nix run .` or remotely with `nix run github:namespace/repo`).